README.md

```diff
@@ -0,0 +1,11 @@
+# WP jQuery Updater for WordPress #
+The development and issue tracker is located here
+
+[![GitHub issues](https://img.shields.io/github/issues/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/issues)
+[![GitHub forks](https://img.shields.io/github/forks/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/network)
+[![GitHub stars](https://img.shields.io/github/stars/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/stargazers)
+[![GitHub license](https://img.shields.io/badge/license-GPLv3-blue.svg)](https://github.com/Remzi1993/wp-jquery-updater/blob/master/LICENSE)
+
+## Bug reports ##
+For bugs, issues use the GitHub [issues](https://github.com/Ramoonus/jQuery-Updater/issues) page.
+If you have any (non bug related) questions, contact me directly
```

Comment:
regel kan weg of link naar support pagina

Reply:
Oké